src/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: #000;
+}
+
+a,
+body,
+h2,
+img {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+
+  margin-bottom: 4px;
+
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.product-card__code {
+  display: inline-block;
+
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+
+  margin: 0 0 17px;
+
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.product-card__feed {
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+  margin-bottom: 16px;
+  align-items: center;
+  line-height: 18px;
+  height: 20px;
+}
+
+.product-card__number {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.product-card__label {
+  font-weight: 400;
+  color: #616070;
+}
+
+.stars {
+  display: flex;
+  width: 60%;
+  justify-content: flex-start;
+}
+
+.star {
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-size: 13px;
+
+  flex-grow: 1;
+
+  width: 17px;
+  height: 17px;
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
What about this checkpoint? 
![image](https://user-images.githubusercontent.com/98764641/206451427-df1d8b09-4d20-44b7-a776-ea1faa559b10.png)